frequently-asked-questions.md

```diff
@@ -0,0 +1,94 @@
+
+## Open Source Programs
+
+**Q: Can previous ambassadors, apply to be an ambassador again?**
+A: Yes! You can re-apply.
+
+**Q: Do GCI/GSoC mentors receive compensation?**
+A: Please email gci-support@google.com or gsoc-support@google.com for inquiries.
+
+**Q: Can I contribute to GSoC projects even if I’m not a GSoC student?**
+A: Yes, GSoC projects are discussed in public channels because we want community involvement. You can even attend weekly meetings of each project if you want to! These projects are just as open source as they were before the GSoC selections. You can contribute with ideas, and even issues if they're not labeled as GSoC issues.
+
+**Q: Do I have to be a part of GCI/GSoC/Ambassadors/Maintainers to contribute to AnitaB.org Open Source?**
+A: No, you don’t need to be a part of those programs to contribute to AnitaB.org Open Source. Anyone can contribute to the open source community anytime.
+
+**Q: Do I have to be a part of Open Source Ambassadors to contribute to AnitaB.org Open Source?**
+A: No, you do not have to be a part of open source teams to contribute to AnitaB.org Open Source.
+
+## Systers to AnitaB.org Open Source change
+
+**Q: What is the difference between Systers Open Source and AnitaB.org Open Source?**
+A: We rebranded from Systers Open Source to AnitaB.org Open Source in 2020. GitHub projects that were active or still needed development have been moved to [github.com/anitab-org](http://github.com/anitab-org). 
+
+**Q: Are we still using Systers-open source on Slack?**
+A: Nope, we moved to Zulip - [AnitaB-org.zulipchat.com](https://anitab-org.zulipchat.com/) and anyone who was in our Slack was invited to the new workspace. We are keeping track of Android Tasks here and Backend Tasks here that were created during Google Code-in 2019.
+
+**Q: Why did we move from Slack to Zulip?**
+A: Here are some reasons:
+- Private Messages were disabled.
+- Age minimum on Slack is 16 years old, which excludes our Google Code-in Participants.
+
+## General 
+
+**Q: Are the Admin 1:1 meetings open to everyone?**
+A: No. The 1:1 Admin meeting is specifically only for admin and invited members. These may happen in accordance with our Code of Conduct.
+
+**Q: Does AnitaB.org Open Source qualify as a Google Developers Group, Women Tech Maker, any other developer community?**
+A: No. AnitaB.org Open Source is a separate open source community which is not qualified for Google Developers Group or Women Tech Maker. Our community mainly focuses on providing opportunities to anyone to get involved in open source projects aligns with AnitaB.org missions and values.
+
+**Q: Where should I post about general questions?**
+A: If you have any questions on where we need help, reach out in #questions or #newcomers or attend one of the open sessions. There are updated in #announcements > Calendar Events.
```

Comment:
There's #general stream too and adding links to them will aslo be better .
Last sentence will be "  **They** are updated in.... "